android/settings.gradle

```diff
@@ -28,48 +28,49 @@ def includeUniversalModule = { name ->
 }
 
 [
-    'expo-ads-admob',
-    'expo-analytics-segment',
-    'expo-app-auth',
-    'expo-app-loader-provider',
-    'expo-av',
-    'expo-background-fetch',
-    'expo-barcode-scanner',
-    'expo-barcode-scanner-interface',
-    'expo-blur',
-    'expo-camera',
-    'expo-camera-interface',
-    'expo-constants',
-    'expo-constants-interface',
-    'expo-contacts',
-    'expo-core',
-    'expo-errors',
-    'expo-facebook',
-    'expo-face-detector',
-    'expo-face-detector-interface',
-    'expo-file-system',
-    'expo-file-system-interface',
-    'expo-font',
-    'expo-font-interface',
-    'expo-gl',
-    'expo-gl-cpp',
-    'expo-google-sign-in',
-    'expo-image-loader-interface',
-    'expo-local-authentication',
-    'expo-localization',
-    'expo-location',
-    'expo-mail-composer',
-    'expo-media-library',
-    'expo-payments-stripe',
-    'expo-permissions',
-    'expo-permissions-interface',
-    'expo-print',
-    'expo-react-native-adapter',
-    'expo-sensors',
-    'expo-sensors-interface',
-    'expo-sms',
-    'expo-speech',
-    'expo-sqlite',
-    'expo-task-manager',
-    'expo-task-manager-interface'
```

Comment:
Reformat this file with Android Studio formatter, it should indent those lines back